Housey pricey.js

```diff
@@ -0,0 +1,42 @@
+// Define the formula for calculating the house price
+const calculateHousePrice = (width, height, depth, gardenSize) => {
+    const volumeInMeters = width * height * depth;
+    return volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+};
+
+// Define Peter's house details
+const peterWidth = 8;
```

Comment:
Variable names can be improved. It's not the width of Peter, but his house's width. `peterHouseWidth`.
It's okay to use lengthy, descriptive variable names, as they can prevent confusion, and can make it unnecessary to explain the code through comments.